proofs/sectorbuilder/rustsectorbuilder.go

```diff
@@ -191,12 +191,6 @@ func (sb *RustSectorBuilder) findSealedSectorMetadata(sectorID uint64) (*SealedS
 			}
```

Comment:
Blocking: I don't have any background in this code so the existing way might be fine.  However the comment on 181 seems to indicate that this whole loop should go away with this change.  As real PIPs are now available.  It seems like this comment should be changed or removed if the loop should still exist even with real PIPs.

Reply:
@ZenGround0 - Good catch. We have:

1. removed the old comment
1. updated the software to correctly marshall PIP and CommP bytes from FFI to Go types
1. added a test (at the SectorBuilder level) which round-trips a CommP and PIP through the verification function